src/main/java/svc/security/HashUtil.java

```diff
@@ -0,0 +1,59 @@
+package svc.security;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component
+public class HashUtil{
+	@Value("${stlcourts.security.municipalitySalt}")
+	String municipalitySalt;
+	@Value("${stlcourts.security.judgeSalt}")
+	String judgeSalt;
+	@Value("${stlcourts.security.courtSalt}")
+	String courtSalt;
+	
+	public Hashids municipalityHashids;
+	public Hashids judgeHashids;
+	public Hashids courtHashids;
+	
+	private void initializeSalts(){
+		if (municipalityHashids == null){
+			municipalityHashids = new Hashids(municipalitySalt,8);
+			judgeHashids = new Hashids(judgeSalt,8);
+			courtHashids = new Hashids(courtSalt,8);
+		}
+	}
+	
+	public String encode(Class<?> fieldClass, long value){
+		initializeSalts();
+		String hashedValue = "";
+		switch (fieldClass.getTypeName()){
```

Comment:
Use getSimpleName instead - that way this code doesn't break if we rename a package

https://docs.oracle.com/javase/8/docs/api/java/lang/Class.html#getSimpleName--